### PR TITLE
ibm5170_cdrom.xml: added Linux flavours

### PR DESCRIPTION
**Pull request by rfka01**
Adding Knoppix 3.1, Kubuntu and Ubuntu flavours.
This comment will be edited as commits come in.

### CODE REVIEW

**Comment by Tafoid** [comment]
@rfka01 Still looking for some of the data (CHDs) you are adding.  Please add them to the others already shared. 
Scanned against latest available ibm5170_cdrom.xml file on this PR.

Missing:
Ubuntu Version 6.06.1 LTS for Intel x86 [folder: ubu6061ltsx86 - size: 0]
missing set: Ubuntu Version 6.06.1 LTS for Intel x86
missing chd: ubuntu_6.06.1_lts_i386.chd [chd-sha1: 14b6cdf1126b5641b482549b04d8b6bbcb98d04c]

Ubuntu Version 7.10 for Intel x86 [folder: ubu710x86 - size: 0]
missing set: Ubuntu Version 7.10 for Intel x86
missing chd: ubuntu_7.10_i386.chd [chd-sha1: 4c825b07bfc7486ec6b1c00a4a9c3343aae649fa]

Ubuntu Version 8.04 LTS Desktop Edition for Intel x86 [folder: ubu804ltsx86desk - size: 0]
missing set: Ubuntu Version 8.04 LTS Desktop Edition for Intel x86
missing chd: ubuntu_8.04_lts_i386_desktop_edition.chd [chd-sha1: 6558b06be0ae4e7539388b47c663b46a4c7199df]

Ubuntu Version 8.10 Desktop Edition for Intel x86 [folder: ubu810x86desk - size: 0]
missing set: Ubuntu Version 8.10 Desktop Edition for Intel x86
missing chd: ubuntu_8.10_i386_desktop_edition.chd [chd-sha1: 55d66617b962c8410c74117032c34cc999025ad8]

Ubuntu Version 9.04 Desktop Edition for Intel x86 [folder: ubu904x86desk - size: 0]
missing set: Ubuntu Version 9.04 Desktop Edition for Intel x86
missing chd: ubuntu_9.04_i386_desktop_edition.chd [chd-sha1: 289cde4dc215bb34c535fb228d90cc947edd5d33]

**Comment by cuavas** [comment]
Also, this is invalid XML – you’re clearly not even testing the changes.

File: hash/ibm5170_cdrom.xml [hunk 1]
```diff
@@ -3742,4 +3742,119 @@ license:CC0
...
+		<description>Ubuntu Version 4.10 for Intel x86 live disk</description>
+		<!-- Warty Warthog, official server https://old-releases.ubuntu.com/releases/4.10/ -->
+		<year>2004</year>
+		<publisher>Canonical Ltd.</publisher>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="warty-release-live-i386" sha1="a28cc349b514b9a50928ce437b6ebbb312c210a8" />
+			</diskarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/ibm5170_cdrom.xml:3784]: You should use parts of a single item for multi-disk releases.

**Comment by rfka01** [comment]
I did the pull request too early, should have finished adding stuff on my side first. Sorry!

**Pull request closed without merging**